momentum/4/lua/ref-msys-tls-params-set.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "msys.tls_params.set"
+description: "msys tls_params set Set a tls parameter string in the Momentum configuration, overriding if the parameter is already set"
+---
+
+<a name="lua.ref.msys.tls_params.set"></a> 
+## Name
+
+msys.tls_params.set — Set a tls parameter string in the Momentum configuration, overriding if the parameter is already set.
+
+## Synopsis
+
+`require('msys.tls_params');`
+
+`msys.tls_params.set(tls_params, "param", "val");`
+
+```
+tls_params: mixed
+param: string
+val: string
+```
+
+## Description
+
+This function sets a tls parameter string in the Momentum configuration, overriding if the parameter is already set.
```

Comment:
I think this is a little confusing - it doesn't really set the config value - it just allows overriding the configured values on a per connection basis right?  Calling this alone will have no effect on configuration.

Reply:
If you call the outbound hook at all, it disregards all values set in the configuration. This is just a way to set them via lua and override what is set in the configuration. Is there a better way to say this? Is adding 'per connection' sufficient?